### DISCUSSION TITLE
Issues Building ElectionGuard for VS w64

### DISCUSSION THREAD

**Discussion by gusmccallum** [microsoft/electionguard#165] [Q&A] [answered]
Hi, I'm trying to build electionguard from the C++ repo to work on it but have been having some issues. When I run the build command I keep getting this error that .cache/CPM isn't found, why would that be? I have all the requirements listed for building on Windows. 

![image](https://user-images.githubusercontent.com/61558904/139946448-4c76b7b8-a38c-4426-881a-ed914e9584b5.png)

**Comment by gusmccallum** [comment, marked as answer]
I don't know how to delete this but I found out the issue was just my path having a space in it.

**Reply by rc-sea** [reply]
thanks for the update i'll close this.

**Comment by rc-sea** [comment]
resolved by user
